static/app/views/replays/replayTable/index.tsx

```diff
@@ -68,6 +69,7 @@ function ReplayTable({fetchError, isFetching, replays, sort, visibleColumns}: Pr
       isEmpty={replays?.length === 0}
       isLoading={isFetching}
       visibleColumns={visibleColumns}
+      data-test-id="replay-table"
```

Comment:
Why the mix of `replay-table` and `replayTable`?

Reply:
I started with `replayTable` then switched to kebab because it seems like the convention.

fixing.